### DISCUSSION TITLE
Testing Phase is now starting!

### DISCUSSION THREAD

**Discussion by sarthakpati** [FETS-AI/Challenge#111] [General]
Hello All,

All who have put their intention for submission via the CMT system should have received a **unique link** to upload their submission. Please ensure you upload related data (**only** the notebook for task-1) by the **26th of July** at **2359 EST**.  

**If you have not received the email link yet, please drop us an email and we will get back to you ASAP!!!**

As a reminder, the manuscript can now be submitted by **2nd of August** at **2359 EST**. Additionally, changes to the parameters used in the notebook (not any algorithmic code) can also be done till the **2nd of August** at **2359 EST**.

Cheers,
Organizing Team

---

UPDATE (2021/08/02): Please use the following for the copyright form:

**Title of the Book or Conference Name:** BrainLes 2020
**Volume Editor(s) Name(s):** Alessandro Crimi, Spyridon Bakas

**Comment by Linardos** [comment]
Is only a single algorithm submission allowed per team or are we allowed to submit a couple?

**Reply by sarthakpati** [reply]
You can submit as many as you want, but each needs to have a separate accompanying paper on CMT and notebook.

**Reply by Linardos** [reply]
Hmm, the thing is these two algorithms wouldn't make sense to appear in different papers. One is optimized for time, the other for performance, but they are based on the same idea. Is it not possible to submit both with this one paper?

**Reply by sarthakpati** [reply]
You could then write it as a single algorithm, which optimizes both at the same time! 😄

**Reply by Linardos** [reply]
Sorry I should have phrased it differently, after trying to optimize both at the same time I ended up with two algorithms in which one inevitably does better on one aspect while the other is doing better on the other. I don't see a clear way to do both at the same time with what I have, sadly. Choosing between the two, I assume performance will be more important correct?

**Reply by sarthakpati** [reply]
> trying to optimize both at the same time I ended up with two algorithms

👏🏽 Very interesting! I can guarantee that all organizers would want to read about the both the approaches! 😄 

> performance will be more important correct

In general, yes, but not if it comes at a significantly larger compute and/or communication costs.

I would recommend that you submit these algorithms independently, and let the ranking system decide which one performs better.

**Reply by eceisik** [reply]
Dear Sarthak,
we implemented a couple of algorithms but we will use just one of them for the challenge and we will mention others on the paper. Yet, do we need to submit all algorithms or the best one is enough?
Thanks,
Ece

**Reply by sarthakpati** [reply]
Hey @eceisik,

You can submit each algorithm as a separate submission or the best one, as you choose. But we would definitely want to read about all of them in the paper. :)

**Reply by eceisik** [reply]
Dear @sarthakpati,

"Additionally, changes to the parameters used in the notebook (not any algorithmic code) can also be done till the 2nd of August at 2359 EST." --> We can change the parameters until August 2 and the changed final version of the parameters will be used for the challenge. Do I understand correctly?  Or the hyperparameters submitted on July 26 will be used for the challenge ranking?
Best,
Ece

**Reply by sarthakpati** [reply]
You understand correctly. No algorithmic changes will be permitted after the 26th of July, but new parameters can be changed till 2nd of Aug.

**Reply by shouldersLL** [reply]
Dear @sarthakpati ,

If multiple submissions are sent to the unique link, will all of them be counted or only the last one will be used for ranking?

Many thanks.

**Reply by sarthakpati** [reply]
I believe files can be overwritten. If not, the newest submission before the deadline will be considered.

**Comment by eceisik** [comment]
Dear Organizers,

our team hasn't received the unique link for the submissions. Could please help us with this problem? I have already sent a separate mail.

Thank you,
Ece

**Reply by sarthakpati** [reply]
@FETS-AI/challenge-organizers this is addressed.

**Reply by eceisik** [reply]
Dear organizers,

can we upload a .py file instead of a jupyter notebook. 

Thanks,
Ece

**Reply by sarthakpati** [reply]
Nope, it needs to be the [main task-1 notebook](https://github.com/FETS-AI/Challenge/blob/main/Task_1/FeTS_Challenge.ipynb) with edits to the specific functions.

**Comment by nlpLover123** [comment]
Dose the parameters change include learning rate or not ? Thanks

**Reply by sarthakpati** [reply]
Yes, it does. It includes **everything** we would need to reproduce your algorithm's results.

**Comment by ghost** [comment]
When will the ranking be released for all participants ?

**Reply by sarthakpati** [reply]
Please see the "Important Dates" section in the website https://fets-ai.github.io/Challenge/

**Comment by ghost** [comment]
Dear organizers,

Can the date for final paper sumbission be extended if we want to submit two independent algorithms? 

Thanks

**Reply by sarthakpati** [reply]
Hi,

Yes, we can provide an extension if you are submitting two independent algorithms.

**Reply by sarthakpati** [reply]
@moonlightarc please drop us an email and we can work out specifics.

**Reply by shouldersLL** [reply]
Dear organizers, @sarthakpati 

How can we change the parameters in the notebook, do we need to resubmit a notebook file? 

Many thanks.

**Reply by sarthakpati** [reply]
Please provide a text file with the specific functions and parameters.

**Reply by shouldersLL** [reply]
Dear organizer, @sarthakpati 

Do we need to submit signed copyright form at this moment? If yes, how can we download it?

Many thanks.

**Reply by sarthakpati** [reply]
You can find the link in https://fets-ai.github.io/Challenge/participate/#participation-timeline, under "short paper submission deadline": https://resource-cms.springernature.com/springer-cms/rest/v1/content/15433008/data/Contract_Book_Contributor_Consent_to_Publish_LNCS_SIP

**Reply by shouldersLL** [reply]
Thank @sarthakpati ,

In the consent form, what should we fill for "Title of the Book or Conference Name:" and "Volume Editor(s) Name(s)".

Many thanks.

**Reply by eceisik** [reply]
Could you also clarify that what should we write for "Volume Editor(s) Name(s)"?

Thanks,
Ece

**Reply by sarthakpati** [reply]
For now, please use the following:

**Title of the Book or Conference Name:** BrainLes 2020  
**Volume Editor(s) Name(s):** Alessandro Crimi, Spyridon Bakas
